code/places-monitor-android/src/phone/java/com/adobe/marketing/mobile/PlacesGeofenceBroadcastReceiver.java

```diff
@@ -55,18 +63,59 @@ public void onReceive(Context context, Intent intent) {
 			return;
 		}
 
-		if (context == null) {
+		GeofencingEvent geofencingEvent = GeofencingEvent.fromIntent(intent);
```

Comment:
GeofencingEvent.fromIntent(intent); can return null you might want to check null first

Reply:
From the[ google docs ](https://developers.google.com/android/reference/com/google/android/gms/location/GeofencingEvent.html#fromIntent(android.content.Intent))
```
Returns
a GeofencingEvent object or null if the given intent is null
```

We do already have a null check on intent. I think we are fine here.
Moreover the google's [sample code](https://developer.android.com/training/location/geofencing) also doesn't have sanity checks. Should be handled well at the receiver level